test/acceptance/test/testsuite_common_test.go

```diff
@@ -40,4 +42,12 @@ var _ = BeforeSuite(func() {
 		WEGO_BIN_PATH = "/usr/local/bin/wego"
 	}
 	log.Infof("WEGO Binary Path: %s", WEGO_BIN_PATH)
+	Expect(checkInitialStatus()).Should(Succeed())
 })
+
+func checkInitialStatus() error {
```

Comment:
Just a minor thing. As this function seems to be related to the testing setup rather than the logic itself. It would be cool to have it in some setup purpose environment, not sure what would that be for Ginko/Gomego yet. But this is the link for the setup function when doing normal testing. https://golang.org/pkg/testing/#hdr-Main . I added an implementation here as well just in case. [link](https://github.com/weaveworks/weave-gitops/blob/main/pkg/gitproviders/common_test.go#L137). Probably something we needed to think about for future work. As as a function like that could be reusable for other suites doing testing for other functions. 🙂